Increase maximum cols/row to 30

BruceGoodGuy opened the pull request:
Hi @timhunt,
In this commit d50ce65ca626e325883ec39224e7ca18c2fcb517,  
I have expanded the maximum of the column/row to 30. I also adjust the CSS and add more CSS classes to the crossword to make the crossword work fine with the new UI.

After discussing with Chris in the ADO the new UI will look like this:

[Admin]
![crossword_new_UI](https://user-images.githubusercontent.com/31382675/195534551-d884ae3f-919b-4bbc-9a5d-ebb6484bf194.png)

[PC]
![crossword_new_UI_2](https://user-images.githubusercontent.com/31382675/195534577-e030ce8d-2953-4272-9f76-b18f4fbf1688.png)

[Mobile]
![crossword_new_UI_3](https://user-images.githubusercontent.com/31382675/195535053-67dcc81f-570b-467b-a9e3-9c315baad2e9.png)

Could you please help me to review it? Thanks.

timhunt commented:
1. Please can you rebase now that the other pull request was merged.
2. It is not essential, but I think I could code getColumnLabel and retrieveCharacterByIndex more elegantly. (Change retrieveCharacterByIndex so 0 -> A - and ensure that the comment makes this clear). Then const first = Math.ceil((i - 1) / 26); and const last = (i - 1) % 26; should work without modification.)
3. Similar in PHP.

Apart from that, this looks fine.

BruceGoodGuy commented:
Hi @timhunt,
In the new commit (d75ed9a74a1a102e6d9bb3659bbab87e673791d4),
I have enhanced the function getColumnLabel. 

From now, when pass the param is 0 the response of the function getColumnLabel will be A, and so on.
I have changed the algorithm a little bit to make it clear. And I added more comments to reviewers easier to read.

Could you please help me to review it again?

timhunt commented:
Thanks. Reviewed and merged.
